pkg/codegen/testing/test/testdata/aws-fargate-pp/dotnet/aws-fargate.cs

```diff
@@ -10,15 +10,15 @@
         Default = true,
     });
 
-    var subnets = Aws.Ec2.GetSubnetIds.Invoke(new()
+    var subnets = vpc.Apply(vpc => Aws.Ec2.GetSubnetIds.Invoke(new()
     {
-        VpcId = vpc.Apply(getVpcResult => getVpcResult.Id),
-    });
+        VpcId = vpc.Id,
+    }));
```

Comment:
This diff doesn't look right. Although it compiles AFAIK, it makes for an ugly result with the outer apply. I think this is because an invoke bound to variable `vpc` was no longer tracked in `generater.functionInvokes` which caused this diff along with the other program diffs in the PR

Reply:
Instead of lowering, I exposed the `token -> schema.Function` mechanism that PCL uses during binding.

Reply:
Nice!